neuro_san/coded_tools/rag_global.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from typing import Any
+from typing import Dict
+from typing import List
+
+from langchain_community.document_loaders import WebBaseLoader
+from langchain_community.vectorstores import InMemoryVectorStore
+from langchain_core.documents import Document
+from langchain_core.vectorstores.base import VectorStoreRetriever
+from langchain_openai import OpenAIEmbeddings
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+
+from neuro_san.interfaces.coded_tool import CodedTool
+
+
+class RAG(CodedTool):
+    """
+    CodedTool implementation which provides a way to utilize different websites' search feature
+    """
```

Comment:
Update the class comment to reflect what this class actually does.
It will be worth spelling out wha the RAG acronym actually means.